### DISCUSSION TITLE
error when opening neovim

### DISCUSSION THREAD

**Discussion by MurraySobol** [neovim/neovim#31807] [General]
Environment: Neovim on Windows 11 (don't laugh!!)
I am getting this message when I launch Neovim:
![image](https://github.com/user-attachments/assets/96451fa9-c7e6-4b26-9cd4-1de831af4107)
When I click Enter it returns to the Home page.
How do I resolve this issue??
Thanks

**Comment by MurraySobol** [comment]
Here is the contents of my init.lua file:
-- Enable Lua language server
vim.cmd('set runtimepath^=~/.vim runtimepath+=~/.vim/after')
vim.cmd('let &packpath=&runtimepath')
lua << EOF
local install_path = vim.fn.stdpath('data') .. '/site/pack/packer/start/packer.nvim'
if vim.fn.empty(vim.fn.glob(install_path)) > 0 then
    vim.cmd('!git clone https://github.com/wbthomason/packer.nvim ' .. install_path)
    vim.cmd('packadd packer.nvim')
end
require('packer').startup(function()
    -- Add your plugins here
end)
EOF

**Reply by zeertzjq** [reply]
You should not use `lua << EOF` in a Lua file.

**Comment by MurraySobol** [comment]
So, the file should look like this??
-- Enable Lua language server
vim.cmd('set runtimepath^=/.vim runtimepath+=/.vim/after')
vim.cmd('let &packpath=&runtimepath')
local install_path = vim.fn.stdpath('data') .. '/site/pack/packer/start/packer.nvim'
if vim.fn.empty(vim.fn.glob(install_path)) > 0 then
vim.cmd('!git clone https://github.com/wbthomason/packer.nvim ' .. install_path)
vim.cmd('packadd packer.nvim')
end
require('packer').startup(function()
-- Add your plugins here
end)
EOF

**Comment by MurraySobol** [comment]
After removing that, I got this message:
![image](https://github.com/user-attachments/assets/aef65a33-b68a-49d5-98cc-e1f9cb6d4772)
